SwrveGoogleSDK/src/main/java/com/swrve/sdk/gcm/SwrveGcmIntentService.java

```diff
@@ -30,14 +22,9 @@ public void onCreate() {
     }
 
     @Override
-    protected void onHandleIntent(Intent intent) {
-        try {
-            handler.onHandleIntent(intent, GoogleCloudMessaging.getInstance(this));
-        } finally {
-            SwrveGcmBroadcastReceiver.completeWakefulIntent(intent); // Always release the wake lock provided by the WakefulBroadcastReceiver.
-        }
+    public void onMessageReceived(String from, Bundle data) {
+        processNotification(data);
```

Comment:
Where is processNotification method? 
Should this be handler.processNotification?